packages/cli/src/commands/dev.ts

```diff
@@ -1,4 +1,6 @@
+import {log} from "@blitzjs/display"
 import {ServerConfig} from "@blitzjs/server"
+import {getConfig} from "@blitzjs/config"
```

Comment:
Change these imports to be dynamic (like `@blitzjs/server` is below). This is so that the code is only parsed and loaded when using this command.

Reply:
Something like this?

```js
try {
  const dev = (await import("@blitzjs/server")).dev
  const {log} = (await import("@blitzjs/display"))
  const {getConfig} = (await import("@blitzjs/config"))
```

Reply:
The first is fine. It's not a network request, so it's fast and doesn't need Promise.all